tests/test_indieauth/test_relmeauth.py

```diff
@@ -9,10 +9,22 @@ def target(self):
 
         return get_valid_relmeauth_links
 
-    def test_rel_me_auth_link_discovery(self, target, index):
-        # NOTE: This test will take a few seconds as it executes multiple requests.
+    def test_rel_me_auth_link_discovery(self, target):
```

Comment:
I think we need to add a `@responses.activate` decorator to this method so we use responses, instead of making a network request.